src/docsfy/main.py

```
@@ -2073,6 +262,36 @@ async def serve_docs(
+# Mount static assets if they exist (production build)
+_assets_dir = _frontend_dist / "assets"
+if _assets_dir.exists():
+    app.mount(
+        "/assets", StaticFiles(directory=str(_assets_dir)), name="frontend-assets"
+    )
+
+
+@app.get("/{path:path}")
+async def spa_catch_all(path: str) -> FileResponse:
+    """Serve the SPA index.html for all non-API, non-docs routes."""
+    logger.debug(f"SPA catch-all serving index.html for path='{path}'")
```

Ruff (0.15.6)

[warning] Logging statement uses f-string

(G004)